cli/flox/src/utils/tracing.rs

```diff
@@ -0,0 +1,20 @@
+use sentry::configure_scope;
+
+/// Sets a tracing tag for the current scope. These always get rolled up to the
+/// root transaction/span but that shouldn't make a difference for searching
+/// and we don't use any child spans at the time of writing anyway.
+///
+/// We use this in place of the following because they aren't searchable in
+/// Sentry:
+///
+/// - `#instrument(fields(foo = "bar"))`
+/// - `Span::current().record("foo", "bar")`
+///
+/// They may support converting fields to tags in future:
+///
+/// - https://github.com/getsentry/sentry-rust/issues/653
+pub fn set_tag<V: ToString>(key: &str, value: V) {
```

Comment:
nit/suggestion: call this `sentry_set_tag` or something alike to show this is a _sentry_ feature, not _some tag_ or tracing related.

Reply:
I did waver about calling this module `sentry` vs `tracing` and whether to use a partially/fully qualified name for the mod::fn to make that clearer. Keeping the module called `tracing` and prefixing the function name with `sentry_` seems like a good balance 👍🏻

Reply:
Done in rebase